testsupport/space/spacerequest.go

```diff
@@ -55,10 +55,14 @@ func WithName(name string) SpaceRequestOption {
 }
 
 func CreateSpaceRequest(t *testing.T, awaitilities wait.Awaitilities, memberName string, opts ...SpaceRequestOption) (*toolchainv1alpha1.SpaceRequest, *toolchainv1alpha1.Space) {
+	hostAwait := awaitilities.Host()
 	memberAwait, err := awaitilities.Member(memberName)
 	require.NoError(t, err)
 	// let's first create a parentSpace
-	parentSpace, _, _ := CreateSpace(t, awaitilities, testspace.WithTierName("appstudio"), testspace.WithSpecTargetCluster(memberAwait.ClusterName))
+	_, mur := testsupport.CreateUserSignupWithSpaceTier(t, awaitilities, "appstudio")
```

Comment:
one thing I just realized - the original code provisioned the space in the given member cluster, the rest of the tests relies on this fact, but the new code would pick any of the member clusters that has enough capacity.
This should be changed so the space is always provisioned to the given member cluster.

Reply:
Good catch!! My bad, sorry! I will address it!